### ISSUE TITLE
'NoneType' object has no attribute 'sigma' in rmgpy.reaction.Reaction.get_mean_sigma_and_epsilon

### ISSUE THREAD

**Issue by mazeau** [cfgoldsmith/RMG-Py#46]
Not sure why this might be, and I have absolutely no idea why changing the thermos would mess this up, so it might have nothing to do with that?

```
/Users/emilymazeau/Code/RMG-Py/rmgpy/rmg/main.py:984: FutureWarning: `rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
  p_min=self.Pmin, p_max=self.Pmax)
Traceback (most recent call last):
  File "/Users/emilymazeau/Code/RMG-Py/rmg.py", line 173, in <module>
    main()
  File "/Users/emilymazeau/Code/RMG-Py/rmg.py", line 167, in main
    rmg.execute(**kwargs)
  File "/Users/emilymazeau/Code/RMG-Py/rmgpy/rmg/main.py", line 940, in execute
    self.check_model()
  File "/Users/emilymazeau/Code/RMG-Py/rmgpy/rmg/main.py", line 984, in check_model
    p_min=self.Pmin, p_max=self.Pmax)
  File "rmgpy/reaction.py", line 1249, in rmgpy.reaction.Reaction.check_collision_limit_violation
  File "rmgpy/reaction.py", line 1271, in rmgpy.reaction.Reaction.check_collision_limit_violation
  File "rmgpy/reaction.py", line 1297, in rmgpy.reaction.Reaction.calculate_coll_limit
  File "rmgpy/reaction.py", line 1297, in rmgpy.reaction.Reaction.calculate_coll_limit
  File "rmgpy/reaction.py", line 1303, in rmgpy.reaction.Reaction.calculate_coll_limit
  File "rmgpy/reaction.py", line 1323, in rmgpy.reaction.Reaction.get_mean_sigma_and_epsilon
  File "rmgpy/reaction.py", line 1323, in rmgpy.reaction.Reaction.get_mean_sigma_and_epsilon
  File "rmgpy/reaction.py", line 1340, in rmgpy.reaction.Reaction.get_mean_sigma_and_epsilon
AttributeError: 'NoneType' object has no attribute 'sigma'
```

**Comment by rwest** [comment]
I guess it is related to the master merge not the new Thermo. Probably they put in a new check that things do not exceed the collision limit (reactions shouldn't be faster than molecules can meet each other) and that requires the Lennard Jones transport properties such as sigma. We probably don't have these for adsorbates, so it's a NoneType. 

We could skip the collision limit check for surface reactions. Or make it ensure sticking coefficient doesn't exceed one, or something, instead. 

Would have to check through the stack trace to see where best to put this.

**Comment by rwest** [comment]
Closed by f6f51c2091e531b0a14968717bb4f2055e391877
